docs/en/query_language/create.md

```diff
@@ -299,4 +299,149 @@ Depending on dictionary [layout](dicts/external_dicts_dict_layout.md) one or mor
 
 For more information, see [External Dictionaries](dicts/external_dicts.md) section.
 
+## CREATE USER {#create-user-statement}
+
+Creates a user account.
+
+ClickHouse allows to configure user accounts in the [user.xml](../operations/settings/settings_users.md) file. The `CREATE USER` statement is an alternative way of creating user accounts.
+
+### Syntax {#create-user-syntax}
+
+```sql
+CREATE USER [IF NOT EXISTS | OR REPLACE] name
+    [IDENTIFIED [WITH {NO_PASSWORD|PLAINTEXT_PASSWORD|SHA256_PASSWORD|SHA256_HASH|DOUBLE_SHA1_PASSWORD|DOUBLE_SHA1_HASH}] BY {'password'|'hash'}]
+    [HOST {LOCAL | NAME 'name' | NAME REGEXP 'name_regexp' | IP 'address' | LIKE 'pattern'} [,...] | ANY | NONE]
+    [DEFAULT ROLE role [,...]]
+    [SETTINGS variable [= value] [MIN [=] min_value] [MAX [=] max_value] [READONLY|WRITABLE] | PROFILE 'profile_name'] [,...]
+```
+
+
+#### Identification
+
+There are some ways of user identification:
+
+- `IDENTIFIED WITH no_password`
+- `IDENTIFIED WITH plaintext_password BY 'qwerty'`
+- `IDENTIFIED WITH sha256_password BY 'qwerty'`
+- `IDENTIFIED WITH sha256_hash BY 'hash'`
+- `IDENTIFIED WITH double_sha1_password BY 'qwerty'`
+- `IDENTIFIED WITH double_sha1_hash BY 'hash'`
+
+To create a user account without a password, use the syntax `IDENTIFIED WITH ... BY`.
+
+The syntax `IDENTIFIED BY ...` creates an identification that uses `sha256_password` or `sha256_hash` based on the value specified after `BY`.
+
+#### User Host
+
+Host can be specified in the `HOST` section of query by the following ways:
+
+- `HOST IP 'IP_ADDRESS_OR_NETWORK'` — User can connect to ClickHouse server only from the specified IP address. For example, `HOST IP '192.168.0.0/16'`.
+- `HOST ANY` — No limits.
+- `HOST LOCAL` — User can connect only locally.
+- `HOST NAME 'FQDN'` —  The user host can be specified as FQDN. For example, `HOST NAME 'mysite.com'`.
+- `HOST NAME REGEXP 'regexp'` — You can use regular expressions when specifying user hosts. For example, `HOST NAME REGEXP '.*\.mysite\.com'`.
```

Comment:
what subset of regexp syntax is supported? (probably determined by regexp library that is used)